packages/cohort_sdk/src/cohort.rs

```diff
@@ -315,32 +380,12 @@ impl Cohort {
                         cause: None,
                     }));
                     db_errors += 1;
-                    false
                 }
-            };
-
-            let rslt_response = result.unwrap();
-            if is_success || self.config.retry_attempts_max <= attempts {
-                break rslt_response;
             }
 
-            match rslt_response {
```

Comment:
debug messages disappeared :)

Reply:
yes, that got cleaned up by mistake. I have added it back with some minor changes